mw-webapp/src/component/accordion/AccordionItem.tsx

```diff
@@ -0,0 +1,39 @@
+import {ReactElement} from "react";
+import {Item as RadixAccordionItem} from "@radix-ui/react-accordion";
+import {AccordionContentProps} from "src/component/accordion/AccordionContent";
+import {AccordionTriggerProps} from "src/component/accordion/AccordionTrigger";
+import styles from "src/component/accordion/AccordionItem.module.scss";
+
+/**
+ * Props for the {@link AccordionItem} component.
+ */
+export interface AccordionItem {
+  /**
+   * React element representing the trigger for this AccordionItem.
+   */
+  trigger: ReactElement<AccordionTriggerProps>;
+  /**
+   * React element representing the content for this AccordionItem.
+   */
+  content: ReactElement<AccordionContentProps>;
+  /**
+   *  A unique key to identify the AccordionItem.
+   */
+  itemKey: string;
+}
+
+/**
+ * A component representing an individual item within the Accordion.
+ * It should be used as a child of the Accordion component.
+ */
+export const AccordionItem = (props: AccordionItem) => {
+  return (
+    <RadixAccordionItem
+      className={styles.accordionItem}
+      value={props.itemKey}
```

Comment:
Is it right that value = itemKey?

Reply:
Yes, RadixAccordionItem must have unique value for the item.
Or would it be better to rename "ItemKey "?

Reply:
No , no need rename